From 14cf21069e18f2b47e0c2acf6526736160f77f98 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Fri, 20 Apr 2018 15:51:52 +0800
Subject: [PATCH 3695/5242] MLK-18096 arm64: dts: imx8qm-mek: drop passthrough
 property for pd_dc1

commit  3839f65c58d74a57f0d54c4484489b9b44214a90 from
https://source.codeaurora.org/external/imx/linux-imx.git

pd_dc1 is a power domain node, marking it as xen,passthrough
meaning disabled. Alought when enabing xen passthrough
the second DC to DomU, we no need this power domain,
but Dom0 linux will disable unused clks and trigger
error message, "populate_gate_pd: failed to get pd".

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Acked-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0.dts     |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index 2b3046b..53cc032 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -171,11 +171,6 @@
 		      <&edma01 0x10>;
 };
 
-/* DC0 */
-&pd_dc1 {
-	xen,passthrough;
-};
-
 &lvds_region2 {
 	xen,passthrough;
 };
-- 
1.7.9.5

